modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/Ratings.js

```diff
@@ -109,4 +112,6 @@ Ratings.propTypes = {
 	url: PropTypes.string.isRequired,
 };
 
-export default Ratings;
+export default function (props) {
```

Comment:
Can we simplify this and keep it the way it was? `export default Ratings;`

Reply:
Yep :/